src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/metrics/CollectBaseDistributionByCycleSpark.java

```diff
@@ -50,11 +50,6 @@
     @Argument(shortName="F", fullName = "pfReadsOnly", doc="If set to true calculate the base distribution over PF reads only.")
     public boolean pfReadsOnly = false;
 
-    @Override
-    public ReadFilter makeReadFilter() {
-        return ReadFilterLibrary.ALLOW_ALL_READS;
-    }
```

Comment:
It seems to me that we still need this override, since the inherited default is `WellformedReadFilter`